docs/Plugins/gitlab.md

```diff
@@ -4,42 +4,71 @@ description: >
   GitLab Plugin
 ---
 
+## Summary
+
+This plugin gathers data from `Gitlab` to display information to the user in `Grafana`. We can help tech leaders answer such questions as:
+
+- Is this month more productive than last?
+- Was our quality improved or not?
+
+## Entities
+
+Check out the [Gitlab entities](/Overview/SupportedDataSources.md#data-collection-scope-by-each-plugin) collected by this plugin.
+
+## Data Refresh Policy
+
+Check out the [data refresh policy](/Overview/SupportedDataSources.md#gitlab) of this plugin.
 
 ## Metrics
 
-| Metric Name                 | Description                                                  |
-|:----------------------------|:-------------------------------------------------------------|
-| Pull Request Count          | Number of Pull/Merge Requests                                |
-| Pull Request Pass Rate      | Ratio of Pull/Merge Review requests to merged                |
-| Pull Request Reviewer Count | Number of Pull/Merge Reviewers                               |
-| Pull Request Review Time    | Time from Pull/Merge created time until merged               |
-| Commit Author Count         | Number of Contributors                                       |
-| Commit Count                | Number of Commits                                            |
-| Added Lines                 | Accumulated Number of New Lines                              |
-| Deleted Lines               | Accumulated Number of Removed Lines                          |
-| Pull Request Review Rounds  | Number of cycles of commits followed by comments/final merge |
+Metrics that can be calculated based on the data collected from Gitlab:
+
+- [Requirement Count](/Metrics/RequirementCount.md)
+- [PR Merge Rate](/Metrics/PRMergeRate.md)
```

Comment:
All these metrics can be calculated. 
1. Explain that PR-related metrics are calculated by GitLab MRs. DevLake uses PR to unify the naming of GitHub PRs, BitBucket PRs, Azure DevOps PRs, and GitLab MRs.
2. Explain that GitLab CI jobs can be used in calculating DORA metrics, if `deployments` are performed through GitLab CI.

![image](https://user-images.githubusercontent.com/14050754/209332525-9defacb1-5c25-417a-8d9c-69c02084835c.png)